packages/components/pie-notification/test/component/pie-notification.spec.ts

```diff
@@ -1,193 +1,161 @@
             });
 
-            test('should render icon-success when variant is success', async ({ mount, page }) => {
+            test('should render icon-success when variant is success', async ({ page }) => {
                 // Arrange
-                await mount(PieNotification, {
-                    props: {
-                        variant: 'success',
-                    },
-                });
+                const notificationPage = new BasePage(page, 'notification');
+                await notificationPage.load({ variant: 'success' });
 
                 // Act
-                const notification = page.locator(componentSelector);
-                const icon = page.locator(headingIconSuccessSelector);
+                const notificationComponent = page.locator(notification.selectors.container.dataTestId);
+                const icon = notificationComponent.getByTestId(notification.selectors.headingIconSuccess.dataTestId);
 
                 // Assert
-                expect(notification).toBeVisible();
-                expect(icon).toBeVisible();
+                await expect(notificationComponent).toBeVisible();
+                await expect(icon).toBeVisible();
             });
 
-            test('should render icon-warning when variant is warning', async ({ mount, page }) => {
+            test('should render icon-warning when variant is warning', async ({ page }) => {
                 // Arrange
-                await mount(PieNotification, {
-                    props: {
-                        variant: 'warning',
-                    },
-                });
+                const notificationPage = new BasePage(page, 'notification');
+                await notificationPage.load({ variant: 'warning' });
 
                 // Act
-                const notification = page.locator(componentSelector);
-                const icon = page.locator(headingIconWarningSelector);
+                const notificationComponent = page.locator(notification.selectors.container.dataTestId);
+                const icon = notificationComponent.getByTestId(notification.selectors.headingIconWarning.dataTestId);
 
                 // Assert
-                expect(notification).toBeVisible();
-                expect(icon).toBeVisible();
+                await expect(notificationComponent).toBeVisible();
+                await expect(icon).toBeVisible();
             });
 
-            test('should render icon-error when variant is error', async ({ mount, page }) => {
+            test('should render icon-error when variant is error', async ({ page }) => {
                 // Arrange
-                await mount(PieNotification, {
-                    props: {
-                        variant: 'error',
-                    },
-                });
+                const notificationPage = new BasePage(page, 'notification');
+                const props: Partial<NotificationProps> = {
+                    variant: 'error',
+                };
```

Comment:
I think all occurrences of this can be changed to the following, as from what I can tell, all props are optional

```suggestion
                const props: NotificationProps = {
                    variant: 'error',
                };
```